### PR TITLE
/firms/search/<term> endpoint by simply proxying to https://api.data.…

### PR DESCRIPTION
**Pull request by catherinedevlin**
…gov/sam/v1

Ready for merge!

Examples to try it out:

http://localhost:8000/api/v1/firms/search/003725520

http://localhost:8000/api/v1/firms/search/intellitech

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@ django-custom-user==0.5
 django-downloadview==1.6
 django-pgjson==0.2.3
 django-rest-auth==0.3.4
+-e git+https://github.com/eofs/django-rest-framework-proxy.git@c4acff85e639d3b9dd15b4fac70c553b003900ac#egg=django_rest_framework_proxy-origin/HEAD
```
Comment (DavidEBest) [inline requirements.txt]: Is there a reason you can't use `django-rest-framework-proxy==1.4.0` instead of HEAD?

Comment (catherinedevlin) [inline requirements.txt]: No, I'll try that next...

File: afsbirez/settings/dev.py [hunk 2]
```diff
@@ -21,4 +22,9 @@
 
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-INSTALLED_APPS.append('django_extensions')
\ No newline at end of file
+INSTALLED_APPS.append('django_extensions')
+
+REST_PROXY = {
```
Comment (DavidEBest) [inline afsbirez/settings/dev.py]: This block is in both the dev and production configuration files. Should it be moved to the base configuration.

Alternatively, we could remove the credentials.py dependency from dev.py by replacing the actual key with `DEMO_KEY`.

Comment (catherinedevlin) [inline afsbirez/settings/dev.py]: That's actually a stopgap until I get a mock version of the SAM API set up.  Once I do, `dev` can lose the credentials entirely and just work against a mock.

**Comment by DavidEBest** [comment]
:+1: Looks good! Can't wait to get it exposed via the front end.

**Pull request merged**